### PR TITLE
serey-w4

### PR DESCRIPTION
**Pull request by MRSEREY**

### CODE REVIEW

File: app/models/enrolment.rb [hunk 1]
```diff
@@ -1,6 +1,13 @@
 class Enrolment < ApplicationRecord
+  before_save :max_students
+
   belongs_to :student
   belongs_to :course
 
   validates :student_id, uniqueness: { scope: [:course_id] }
+  validate :max_students
+
```
Comment (kimsrung) [inline app/models/enrolment.rb:9]: no validation on course. error when create enrollment without selecting any course

File: config/routes.rb [hunk 2]
```diff
@@ -3,6 +3,7 @@
   
   resources :courses
   resources :teachers
+  resources :enrolments
```
Comment (kimsrung) [inline config/routes.rb:6]: wrong spelling 'enrollments'

Comment (MRSEREY) [reply config/routes.rb:6]: https://www.google.com/search?ei=-nXjXITnN5q89QOrlqTQCA&q=enrollment&oq=enrollment&gs_l=psy-ab.3..0l4j0i30l6.54854.54854..55759...0.0..0.81.81.1......0....1..gws-wiz.......0i71.42T2JPksxdI

Comment (MRSEREY) [reply config/routes.rb:6]: I follow UK

**Pull request closed without merging**